events/2024-06-10-mtb-ngs.html

```diff
@@ -14,15 +14,15 @@
 date_end: 2024-06-14 # optional, if event is more than one day
 
 contributions:
-  organisers: # GTN contributors or funders, must be defined in CONTRIBUTORS.yaml
-  - dbrites
-  instructors:
+  organisers &  instructors: # GTN contributors or funders, must be defined in CONTRIBUTORS.yaml
   - dbrites
   - cstritt
   - andreacabibbe
   - arashghodousi
   - federicodimarco
   - pvanheus
+  - Liliana Rutaihwa
+  - Linzy Elton
```

Comment:
These two contributors will have to be added to the CONTRIBUTORS.yaml file. I will do that for you, just let me know if they have github accounts, and what their affilliations are (not mandatory, but we can add that if you want)